shm/alcohol.cpp

```diff
@@ -0,0 +1,17 @@
+#include "alcohol.hpp"
+
+Alcohol::Alcohol(std::string name, size_t amount, size_t alcoholByVolume)
+    : Cargo(name, amount, alcoholByVolume_){};
+
+bool Alcohol::operator==(Cargo &cargo) const {}
+
+size_t Alcohol::getPrice() const {
+  if (alcoholByVolume_ < maxAlcoholByVolume_) {
+    return basePrice_ * alcoholByVolume_ / maxAlcoholByVolume_;
```

Comment:
If this 2 value will be equal you get `1`. So `if` is redundant. 
You also need to cast it to double/ float, because you loose parts after the decimal point